app/components/summary_list_component.rb

```diff
@@ -22,4 +40,8 @@ def transform_hash(row_hash)
       }
     end
   end
+
+  def any_row_has_action_span?
+    rows.select { |row| row.key?(:action) }.any?
+  end
```

Comment:
I don’t think we need this anymore, as the summary list component performs this check.

```suggestion
```

Reply:
Ah great, will remove

Reply:
Hmmm, scrap that; removing didn’t work! I think it’s because the component is looking for `present?`, not `any?`:

https://github.com/DFE-Digital/govuk-components/blob/0b39f48f3745748a09d3c179a5418031ed40f3eb/app/components/govuk_component/summary_list.rb#L7-L9

This’ll need refactoring when moving to v2.0.0 of `govuk-components`, as that handles actions in a much better but also quite different way. So let’s leave it as it is.